bin/live/pycbc_live_combine_single_fits

```diff
@@ -132,23 +133,15 @@ fout['bins_edges'] = list(bl) + [bu[-1]]
 fout['fits_dates'] = ad + start_time_n
 
 for ifo in args.ifos:
+    logging.info(ifo)
     fout.create_group(ifo)
```

Comment:
`fout_ifo = fout.create_group(ifo)` will avoid a line - I cant see a way to suggest this